docs/examples/cookbook/drag-and-drop.md

```diff
@@ -296,7 +296,7 @@ local allEntries = scope:ForValues(
 			},
 			Dragging = {
 				MousePosition = mousePos,
-				SelfDragInfo = scope:Computed(function(use)
+				SelfDragInfo = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
				SelfDragInfo = scope:Computed(function(use)
```

Reply:
reverted